go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cloudhut/kminion/v2
 
-go 1.23
+go 1.23.0
```

Comment:
Is that needed? I think Go releaser will use v1.23.0 and not latest v1.23 in that case

Reply:
We can bump that to 1.24 FWIW

Reply:
Missed that, I think `go get` added the patch number